lib/plugins/aws/package/compile/events/alb/lib/target-groups.js

```diff
@@ -83,7 +84,10 @@ module.exports = {
             },
           ],
         },
-        DependsOn: [registerTargetPermissionLogicalId],
+        DependsOn: [
+          registerTargetPermissionLogicalId,
+          _.get(functionObj.targetAlias, 'logicalId'),
+        ].filter((d) => d),
```

Comment:
Let's use `filter(Boolean)` (it's more straightforward to read the intention)

Reply:
Ok, updated